### DISCUSSION TITLE
What do the green and red highlights mean? If something is green/red, is that a good thing or a bad thing?

### DISCUSSION THREAD

**Discussion by 11BelowStudio** [zodiacon/PoolMonXv3#1] [Q&A] [answered]
Recently downloaded this program to help diagnose a memory leak (I think I found the driver responsible for it, currently monitoring PoolMonXv3 to see if the fix worked), but I have a question about the red/green highlights that keep appearing and disappearing - what exactly do they mean?

Do the numbers go green when going up and red when going down or something like that? Or  do the colours denote something else (like, whether or not the number is a good number or a bad number)?

The README doesn't mention anything about it, nothing in the program mentions what they mean, and I doubt my ability to work it out from the source code. So, if anyone does know what the colours actually mean, please could you let me know? Thanks!

**Comment by zodiacon** [comment, marked as answer]
Yes, the green means number is up, and red means number is down (from the previous refresh).
I will add that to the Readme.
